### PR TITLE
Chainlink example project included

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by deverka — COMMENTED** [review]
Looks good to me. Thanks @bgodlin 
@jamesbayly can you please take a look in a spare moment?

File: Ethereum/ethereum-chainlink/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "declaration": true,
+    "importHelpers": true,
+    "resolveJsonModule": true,
+    "module": "commonjs",
+    "outDir": "dist",
+    "rootDir": "src",
+    "target": "es2017"
```
Comment (jamesbayly) [inline Ethereum/ethereum-chainlink/tsconfig.json:12]: its good practice to enable `strict: "true"` here. It exposes some other bugs etc

Comment (bgodlin) [reply Ethereum/ethereum-chainlink/tsconfig.json]: This is resolved in https://github.com/subquery/ethereum-subql-starter/pull/39/commits/a892fd39b2a620b1465afd667a87ce0b2fe4fb16

**Comment by jamesbayly** [comment]
You are missing the `.git` directory, otherwise it's fine by me

**Review by jamesbayly — APPROVED** [review]

**Comment by bgodlin** [comment]
> You are missing the `.git` directory, otherwise it's fine by me

I cannot find the `.git` directory in any project folder except for the root directory, as expected

**Comment by jamesbayly** [comment]
Sorry, `.github`, i've added it

**Pull request merged**